File: 2012/November/20-11-2012/string_calculator_spec.rb
require_relative 'string_calculator'

describe 'StringCalculator' do
  let(:calc) {StringCalculator.new}
	describe "#add" do
		it 'returns 0' do
      calc.add("").should == 0
		end

    it 'returns 3' do
      calc.add("1,2").should == 3
    end

	end
end

File: 2012/November/23-11-2012/spec/StringCalculatorSpec.js
describe("StringCalculator", function() {
  var stringCalculator;

  beforeEach(function() {
    stringCalculator = new StringCalculator();
  })

  it("should return 0 given empty string", function() {
    expect(stringCalculator.add("")).toEqual(0);
  });

  it("should return 1 given '1'", function() {
    expect(stringCalculator.add("1")).toEqual(1);
  });
})